packages/prosess-vedtak/src/components/brev/MellomLagreBrev.tsx

```diff
@@ -1,255 +0,0 @@
-import React, { useEffect, useState } from 'react';
```

Comment:
Mellomlagringsfunksjonen utgår da det nå håndteres for de aller fleste tilfellene inne i editoren, med autolagring.